crawler.py

```diff
@@ -23,6 +23,8 @@
 import filterpaths
 import generatehtml
 
+not_clickable_elems = set()
```

Comment:
We should create the set in do_something and reset it as soon as we perform an action.
For example, let's say there's a popup window A that is hiding an element B. At the beginning we will try to click B and add it to the set because it's hidden by A. Then we will close A. Now if the set is global, we will not click on B anymore, even though it is now clickable.